docs/it/aspetti-finanziari-i-soldi.md

```diff
@@ -0,0 +1,279 @@
+# Aspetti Finanziari "I Soldi"?
+{: .no_toc }
+
+- TOC
+{:toc}
+
+<!-- prettier-ignore-end -->
+
+Il settore delle tecnologie dell’informazione e della comunicazione, anche noto con gli acronimi ICT
+o TIC, è uno degli ambiti professionali con tassi di crescita a ritmi esponenziali negli ultimi anni,
+grazie alla velocità con cui organizzazioni e aziende innovano e, soprattutto, in virtù del processo di
+digitalizzazione, che sta toccando ogni settore della società contemporanea. In virtù di questa
+evoluzione il settore ad oggi è tra quelli che permettono un più veloce inserimento nel mondo del
+lavoro, soprattutto tra i giovani neolaureati, con buoni risultati anche a livello economico,
+molteplici le figure professionali ricercate, tra le più richieste sicuramente:
+
+- Amministratore di database;
+- Cybersecurity analyst;
+- Web developer;
+- Software developer;
+- Cloud architect;
+- Amministratore di sistema;
+- IT manager;
+- Analista di sistema;
+- Tecnico Help Desk;
+
+tante figure professionali che permettono anche una discreta scelta nella tipologia di attività da
+svolgere in riferimento alle proprie capacità e desideri professionali. Chi si approccia a questo
+mondo sicuramente deve avere una propensione di base all’ utilizzo di tutti gli strumenti
+informatici nonché una discreta dose di costanza, dedizione e passione. Quindi il primo passo da
+compiere per chi si affaccia al mondo dell’ IT e senz’altro la scelta del tipo di lavoro che si intende
+svolgere e anche le modalità ovvero se operare in virtù di un contratto di lavoro dipendente o
+come sviluppatore freelance, senz’ altro la seconda scelta implica delle difficoltà aggiuntive come
+ogni attività imprenditoriale in quanto si devono fronteggiare incertezza e difficoltà economica,
+almeno all’ avvio dell’ attività, e ricercare da solo i nuovi clienti nonché scegliere i progetti da
+approcciare facendo attenzione che siano calibrati sia alle capacità professionali acquisite nonché
+alle dimensioni del progetto stesso. Ovviamemente se la scelta di operare come freelance da un
+lato porta maggiori difficoltà e responsabilità dall’ altro può condurre chi sceglie questa strada a
+importanti risultati sia in termini reddituali ma anche in termini di soddisfazione professionale
+poiché sarai libero di organizzare il tuo tempo e scegliere su cosa lavorare tutti aspetti certamente
+da non sottovalutare. A conclusione di questa breve introduzione non possiamo far altro che dire
+che il primo passo è farti delle domande per capire se questo settore è quello giusto per te e se sì
+cosa vuoi ottenere, quindi se hai scelto di entrare in questo ambito professionale leggi il seguito e
+ti daremo delle utili informazioni sia che tu ti voglia approcciare con un contratto di lavoro
+dipendente o come freelance.
+
+## Freelance: prestazione occasionale o partita iva?
+
+Con la prestazione occasionale si ha la possibilità di testare se stessi da freelance e ad onor del
```

Comment:
> Con la prestazione occasionale si ha la possibilità di testare se stessi da freelance e ad onor del
vero è il principale strumento utilizzati di comincia in quanto di semplice attuazione 

`il principale strumento utilizzati di comincia` credo sia questo il punto in cui mi sono partite le ventole.
Aggiungerei anche qualche virgola tipo: e, ad onor del vero, ...